noxfile_config.py

```diff
@@ -32,6 +32,8 @@
     # to use your own Cloud project.
     "gcloud_project_env": "GOOGLE_CLOUD_PROJECT",
     # 'gcloud_project_env': 'BUILD_SPECIFIC_GCLOUD_PROJECT',
+    # If you need to use a specific version of pip, change _pip_version_override to the string representation of the version number, for example, "20.2.4"
```

Comment:
Lint _might_ complain saying this is too long :C

Reply:
haha I'm fixing it now even though it didn't complain

Reply:
Sounds good! XD